oem/ibm/configurations/pdr/ibm,rainier-2u/11.json

```diff
@@ -1795,6 +1795,91 @@
              }
         }]
     },
+    {
+        "entity_path" : "/xyz/openbmc_project/inventory/system/chassis/motherboard/pcieslot10/pcie_card10/cxp_top",
+        "effecters" : [{
```

Comment:
I thought the cxp_top and bottom are removed ?

Reply:
I discussed with santhosh....cxp top/bot are IBM specific and connector 0..3 are flett specific and we need to model both. cable card port can have any of those..